wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -207,8 +207,10 @@ impl PhysicalDeviceFeatures {
             | F::PUSH_CONSTANTS
             | F::ADDRESS_MODE_CLAMP_TO_BORDER
             | F::SAMPLED_TEXTURE_BINDING_ARRAY
+            | F::BUFFER_BINDING_ARRAY
             | F::STORAGE_TEXTURE_BINDING_ARRAY
-            | F::BUFFER_BINDING_ARRAY;
+            | F::BUFFER_BINDING_ARRAY
```

Comment:
buffer binding array is mentioned twice now

Reply:
fixed